cfgov/unprocessed/css/forms.less

```diff
@@ -606,26 +608,51 @@
     margin-bottom: unit(30px / @base-font-size-px, em);
 }
 
+.form-label-header + .form-l .form-label-header {
+    // Override heading-5() settings.
+    .webfont-regular();
+    font-weight: normal;
+    text-transform: none;
+    margin-bottom: unit(10px / @base-font-size-px, em);
+}
+
+.form-l {
+    .respond-to-min(@bp-sm-min, {
+        .grid_nested-col-group();
+    });
+}
+
+.form-l_col input{
+    box-sizing: border-box;
+    width: 100%;
+}
+
+.form-l_col-1-4 {
+    .respond-to-min(@bp-sm-min, {
+        .grid_column(1, 4);
+    });
+}
+
 .form-l_col-1-2 {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/d0bd47229a4c3b4b8cb2f1400c6924a3fc7e5e0d